File: monolitico/spa/src/main/java/br/com/unialfa/pos/soa/spa/core/model/entity/Comentario.java
package br.com.unialfa.pos.soa.spa.core.model.entity;

import java.util.Date;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.JoinColumn;
import javax.persistence.OneToOne;
import javax.persistence.Table;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Entity
@Table(name = "comentario")
@Data
@EqualsAndHashCode(callSuper = false)
public class Comentario extends AbstractEntity {
	@Column
	private String corpo;

	@Column
	private Date data;

	@OneToOne
	@JoinColumn(name = "usuario_id", nullable = false)
	private Usuario autor;

	@OneToOne
	@JoinColumn(name = "tarefa_id", nullable = false)
	private Tarefa tarefa;
}

File: projetos_alunos/soa-alex-arthur-edipo-felipe/view-soa-master/src/models/comentario.ts
import { Alocacoes } from "./alocacoes";
import { Aluno } from "./aluno";
import { Tarefa } from "./tarefa";

export class Comentario {

    constructor() { }

    public Id: number;
    public Tarefa: Tarefa;
    public Aluno: Aluno;
    public Comentario: string;
    public Data: Date;
}